cookbook/core/type_system/structured_dataset.py

```diff
@@ -50,8 +50,8 @@
 
 
 # %%
-# We define two tasks — one returns a pandas DataFrame, and the other, a :py:class:`flytekit.types.schema.FlyteSchema`.
-# The DataFrames themselves will be serialized to an intermediate format like parquet before sending them to the other tasks.
+# We define two tasks— one returns a pandas DataFrame, and the other, a :py:class:`flytekit.types.schema.FlyteSchema`.
+# The DataFrames will be serialized to an intermediate format like parquet before sending them to the other tasks.
```

Comment:
Passive. Who or what is serializing to an intermediate format?

Typo: "parquet". Should this be "Apache Parquet" or "Parquet"? Ref: https://parquet.apache.org/
 
Ambiguous, out of scope: "like parquet". What does this sentence describe, the action or the format? Choose one. Move the format to another sentence. Something like "[actor] serializes the DataFrames to an intermediate format before sending them.... This intermediate format is Parquet."

Reply:
I think we mean parquet file. Would this look good- "The DataFrames are serialized to an intermediate format (like parquet file) before sending them to the other tasks."?